test/integration/cli-domain-package-server-script/cli-domain-package-server-script.js

```diff
@@ -453,15 +456,7 @@ describe('cli : domain : package-server-script', function(){
 
             it('should wrap while/do/for;; loops with an iterator limit', function(){
               var compiledContent = fs.readFileSync(path.resolve(publishPath, result.src), {encoding: 'utf8'});
-              expect(compiledContent).to.contain('var __ITER = 1000000000;while (true) { if(__ITER <=0){ throw new Error' +
-              '("Loop exceeded maximum allowed iterations"); } \n' +
-              '    x = 234;\n   __ITER--; }');
-              expect(compiledContent).to.contain('var __ITER = 1000000000;for (var i = 1e12;;) { if(__ITER <=0){ throw new Error' +
-              '("Loop exceeded maximum allowed iterations"); } \n' +
-              '    y = 546;\n   __ITER--; }');
-              expect(compiledContent).to.contain('var __ITER = 1000000000;do { if(__ITER <=0){ throw new Error' +
-              '("Loop exceeded maximum allowed iterations"); } \n' +
-              '    z = 342;\n   __ITER--; } while (true);');
+              expect(compiledContent.match(/Loop exceeded maximum allowed iterations/g).length).to.equal(3);
```

Comment:
Here it became complicated to test the exact replacement as it gets minified after that. In any case, I test that there are 3 replacements containing that string, which is the number of loops I expect. Good enough, I guess :rofl: